bigframes/geopandas/geoseries.py

```diff
@@ -39,3 +40,9 @@ def y(self) -> bigframes.series.Series:
         series = self._apply_unary_op(ops.geo_y_op)
         series.name = None
         return series
+
+    @property
+    def area(self, crs=None) -> bigframes.series.Series:
```

Comment:
Let's try this to avoid the mypy failure:

```suggestion
    # GeoSeries.area overrides Series.area with something totally different.
    # Ignore this type error, as we are trying to be as close to geopandas as
    # we can.
    @property
    def area(self, crs=None) -> bigframes.series.Series:  # type: ignore
```